### DISCUSSION TITLE
Change output file name?

### DISCUSSION THREAD

**Discussion by SawkeeReemo** [cross-seed/cross-seed#253] [Q&A] [answered]
Is there a way to get the output torrent file name to change so that the torrent name is first instead of the type and tracker name? I'd love to have these sorted by name. Possible?

**Comment by mmgoodnow** [comment, marked as answer]
There is not.

**Reply by SawkeeReemo** [reply]
Cool. No worries! Using inject make this a moot point anyway. Thanks!
